lib/event/event.rb

```diff
@@ -188,4 +188,14 @@ def ==(other)
       other.half_day_at_start == half_day_at_start &&
       other.half_day_at_end == half_day_at_end
   end
+
+  def to_h
```

Comment:
I'm sorry. I take it back. This probably is a `as_json`, since we also stringify the dates.